### PR TITLE
Add more accessibility question

### PR DESCRIPTION
**Pull request by jayantpranjal0**
Add question on autocomplete attribute

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

On issue #374 

Thank You!

### CODE REVIEW

File: src/data/accessibility-quiz.ts [hunk 1]
```diff
@@ -257,6 +257,15 @@ const accessibilityQuiz = [
...
     Link: "https://www.freecodecamp.org/news/react-accessibility-tools-build-accessible-react-apps/#:~:text=Why%20you%20should,from%20the%20beginning%3A",
   },
+  {
+    Question: "Among the following options where can autocomplete attribue be not used?",
+    Answer:"<html>",
+    Distractor1:"<input>",
+    Distractor2:"<text>",
+    Distractor3:"<form>",
+    Explanation:"Autocomplete can be used on <input>, <textarea>, <select> and <form> elements.",
+    Link:"https://www.w3schools.com/accessibility/accessibility_autocomplete.php",
```
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:267]: Please only use freeCodeCamp resources or MDN

File: src/data/accessibility-quiz.ts [hunk 2]
```diff
@@ -257,6 +257,15 @@ const accessibilityQuiz = [
       "If you don't pay attention to accessibility right from the beginning of your project, it may become an expensive burden once you start retrofitting.",
     Link: "https://www.freecodecamp.org/news/react-accessibility-tools-build-accessible-react-apps/#:~:text=Why%20you%20should,from%20the%20beginning%3A",
   },
+  {
+    Question: "Among the following options where can autocomplete attribue be not used?",
```
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:261]:
```suggestion
    Question: "Which of the following HTML tags does NOT use the autocomplete attribute?",
```

File: src/data/accessibility-quiz.ts [hunk 3]
```diff
@@ -257,6 +257,15 @@ const accessibilityQuiz = [
...
+    Answer:"<html>",
+    Distractor1:"<input>",
+    Distractor2:"<text>",
+    Distractor3:"<form>",
+    Explanation:"Autocomplete can be used on <input>, <textarea>, <select> and <form> elements.",
```
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:266]:
```suggestion
    Explanation:"The autocomplete attribute can be used in the following HTML elements: <input>, <textarea>, <select> and <form>.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by jayantpranjal0** [comment]
Thank you for your suggestions. I have made the changes as asked for the PR to be merged.

File: src/data/accessibility-quiz.ts (hunk 1 above)
Comment (jdwilkin4) [reply src/data/accessibility-quiz.ts:267]: @jayantpranjal0 

This one still needs to be addressed

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**